src/Network/DO/Tags/Net.hs

```diff
@@ -0,0 +1,101 @@
+{-# LANGUAGE DoAndIfThenElse #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+-- | Network interpreter for Tags specific API
+module Network.DO.Tags.Net(tagsCommandsInterpreter) where
+
+import Prelude                      as P hiding (error)
+
+import Control.Comonad.Env.Class    (ComonadEnv, ask)
+import Control.Monad.Trans          (MonadIO)
+import Data.Aeson                   as A hiding (Result, pairs)
+import Data.Proxy
+import Network.DO.Tags.Commands
+import Network.DO.Net.Common
+import Network.DO.Types             as DO hiding (URI, name)
+import Network.REST
+
+
+-- Define some shortcut accessor
+tagsURI :: String
+tagsURI = "tags"
+
+tagsEndpoint :: String
+tagsEndpoint = rootURI </> apiVersion </> tagsURI
+
+
+-- Derive the listTags endpoint for free, see @Network.DO.Net.Common@
+--
+-- https://developers.digitalocean.com/documentation/v2/#list-all-tags
```

Comment:
Well, I thought about it but in those *.Net, only the interpreter is exported. Hence, having haddock comments for other parts is kinda useless yet harmless. It doesn't cost a lot to turn them as haddock such that if a function is exported in the future, a comment is already there.